packaging/debian-sid/snapd.lintian-overrides

```diff
@@ -3,17 +3,17 @@ snapd: non-standard-dir-in-var [var/snap/]
 snapd: non-standard-toplevel-dir [snap/]
 # The Exec= line is /bin/false and the whole thing relies on bus activation.
 snapd: desktop-command-not-in-package bin/false [usr/share/applications/io.snapcraft.SessionAgent.desktop]
-# Snapd uses setuid-root helper to launch programs.
-snapd: elevated-privileges 4755 root/root [usr/lib/snapd/snap-confine]
```

Comment:
oh, caps don't count as elevated privileges?

Reply:
Caps are not in the actual package. It's silly but true.